server/index.js

```diff
@@ -86,12 +86,15 @@ function normalizeContentPath(start) {
 }
 
 app.get("/_open", (req, res) => {
-  const filepath = req.query.filepath;
+  const { filepath } = req.query;
   if (!filepath) {
     throw new Error("No .filepath in the request query");
   }
-  openEditor([filepath]);
-  res.status(200).send(`Tried to open ${filepath} in ${process.env.EDITOR}`);
+  const absoluteFilepath = normalizeContentPath(filepath);
```

Comment:
THIS is the most important change. It used to be that the builder would make sure this query string was something like `/Users/peterbe/dev/MOZILLA/MDN/stumptown-renderer/content/files/en-us/web/html/element/video/index.html` but now it's `content/files/en-us/web/html/element/video/index.html`